src/pages/host/getHostEventString.tsx

```diff
@@ -32,9 +32,20 @@ export const getHostEventString = (
   data: HostEventLogData
 ) => {
   const succeededString = "succeeded";
+  const failedString = "failed";
+
   switch (eventType) {
     case HostEvent.Created:
-      return <span data-cy="created">Host created</span>;
+      return (
+        <div data-cy="created">
+          Host creation {data.successful ? succeededString : failedString}
+          {data.logs ? (
+            <HostEventLog title="Attached logs" logs={data.logs} isCode />
```

Comment:
What do you think about sticking with "Additional details"? I think "attached" doesn't have much meaning for users and is maybe a bit confusing given the other titles like "Scripted logs" and "Provisioning logs".